source/_introduction.html.md.erb

```diff
@@ -298,9 +298,37 @@ The JSON representation of the objects returned from the List All REST endpoints
 Smartsheet will add new functionality and bug fixes to the API over time. Make sure that your code can handle new JSON properties gracefully. 
 Also, make sure your code does not depend on the order in which JSON objects are returned, unless it is explicitly stated in this documentation. 
 
+In some cases, such as when new column types or new widgets are added, Smartsheet offers a backwards-compatibile option as well as giving you access to the new functionality. For instance, if a sheet has column types of MULTI_CONTACT_LIST or MULTI_PICKLIST, there are two ways of receiving the response:
+
+* As text, such as column type TEXT_NUMBER
+* As a complex object returning the data in the most convenient form for that column type, such as MULTI_CONTACT_LIST or MULTI_PICKLIST
+
+Smartsheet uses two indicators to help you discover changes in the return data for your API calls:
+
+* level: a query parameter that is incremented when a new feature or column type is added
+* version: a read-only attribute in the Column object that is incremented when a new column type is added
+
+You must use the level query parameter, for example **?level=2**, to return a complex object with the new column type. Without the query parameter, the response will be backwards-compatible (in most cases, a string).
+
+Use the highest possible level for each endpoint, as in the following table:
+
+Endpoint category | Level
+-----|-----|
+Sights (aka dashboards) | 4
+Reports | 3 
+Sheets | 2 
+
+The read-only "version" attribute for the Column object increments when a new column type is added:
+
+Multi-contact | Multi-picklist
+-----|-----|
+1 | 2 
```

Comment:
what does this table represent? 

If I do a GET on a multipicklist column and use level = 1 , then what column "version" do i get?

Reply:
Theoretically, you still get version = 2, but the column type is TEXT_NUMBER or whatever. I'm not sure.

Reply:
Can you verify this? I have a feeling that it will be version 1!

Reply:
Version is independent of the level param. i.e., you get version=2 whether you use a level param or don't. The column type is thing that changes, and then the way the object is formatted.